src/containers/Application/ApplicationOverview.tsx

```diff
@@ -55,14 +70,22 @@ const ApplicationOverview: React.FC = () => {
     <Loading />
   ) : submitError ? (
     <Message error header="Problem to submit application" list={[submitError]} />
-  ) : serialNumber && elementsInSections ? (
+  ) : serialNumber && appStatus && sectionsPages ? (
     <Container>
-      <ApplicationSummary
-        sectionsAndElements={elementsInSections}
-        onSubmitHandler={() => submit()}
-        appStatus={appStatus}
-      />
-      {showProcessingModal(processing, submitted)}
+      <Header as="h1" content="REVIEW AND SUBMIT" />
+      <Form>
+        {sectionsPages.map((sectionPages) => (
+          <SectionSummary
+            sectionPages={sectionPages}
+            serialNumber={serialNumber}
+            isDraft={appStatus.status === 'DRAFT'}
```

Comment:
I was only removing the **logic** from that component. The presentation component would only know `what` to display and not what are the rules to show that part of the component. But I agree that we would have more different checking before we load each `SectionSummary` that would be dealt by the `ApplicationOverview` or even other components/containers.

Reply:
Sure, so maybe the field should be called something more like `applicantCanEdit`, because as you said, the child component on needs to know what to show, not how that value was deduced. So later if the condition becomes more complex (e.g. `applicantCanEdit={appStatus.status === 'DRAFT' || appStatus.status === 'CHANGES.REQUIRED'`} then we don't have to worry about the child component prop being confusingly named `isDraft`.

Reply:
Yep! Good point : ) Renaming this now.

Reply:
Updated